templates/hound.yml

```diff
@@ -1,17 +1,17 @@
 # See https://houndci.com/configuration for help.
 coffeescript:
-  # config_file: .coffeescript-style.json
+  config_file: .coffeescript-style.json
```

Comment:
Should this one remain commented out; is there no config we are pulling from thoughtbot/guides, as it doesn't exist?